woocommerce/woocommerce-admin
# Store Profiler - include Creative Mail as a free extension

Fixes #5288

Adding Creative Mail as free extension in the mail flow of the onboarding wizard. This resolves only the first requirement of the linked issue.

### Screenshots

![image](https://user-images.githubusercontent.com/444632/98308619-586f6300-1f7d-11eb-941f-ae652042e8f1.png)

### Detailed test instructions:

- Check out branch
- Navigate to Onboarding wizard
- Proceed to business-details step
- Confirm that the Creative Mail option is present and matches screenshot
- Complete wizard
- Ensure plugin is installed
